src/components/graph-vis/helpPanel.js

```diff
@@ -0,0 +1,320 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+
+const useStyles = makeStyles(theme => ({
+  typography: {
+    padding: theme.spacing(1),
+  },
+}));
+
+export function HelpPanel({name, children}) {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleClick = event => {
+    setAnchorEl(event.currentTarget);
+    event.stopPropagation();
+    event.preventDefault();
+  };
+
+  const handleClose = event => {
+    setAnchorEl(null);
+    event.stopPropagation();
+    event.preventDefault();
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? name : undefined;
+
+  return (
+    <>
+      <FontAwesomeIcon
+        icon={faQuestionCircle}
+        onClick={handleClick}
+      />
+      <Popover
+        id={id}
+        style={{
+          maxWidth: '400px',
+        }}
+        open={open}
+        anchorEl={anchorEl}
+        onClick={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}>
+          <Typography className={classes.typography}>
+            {children}
+          </Typography>
+      </Popover>
+    </>
+  );
+}
+
+export function HelpPanelLabel({name, label, children}) {
+  return (
+    <span>
+      {label}
+      &nbsp;
+      <HelpPanel
+        name={name}
+        children={children}
+      />
+    </span>
+  );
+}
+
+export function SearchHelp() {
+  return (
+    <HelpPanelLabel name='search-help' label='Search'>
+      <p>
+        Use this panel to search for items in the loaded dataset. Search results
+        will match the phrase used anywhere in the name of the item. For
+        example, searching for <tt>N</tt> will match Nitrogen (<tt>N</tt>), but
```

Comment:
tt tag is obsolete: 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt

Reply:
what about: strong, i, pre, code?

Reply:
See b81bef379e3906c69b45bd2149702c5459f7cbb6. How about `<samp>`? :smile_cat: